src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -0,0 +1,92 @@
+package com.minecolonies.coremod.commands.generalcommands;
+
+import com.minecolonies.api.colony.permissions.Rank;
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.AbstractSingleCommand;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+public class CheckForAutoDeletesCommand extends AbstractSingleCommand
+{
+
+    public static final  String DESC              = "check";
+    private static final String NO_COLONIES_FOUND = "There where no colonies found that could be auto-deleted";
+    private static final String ABLE_TO_DELETE    = "We were able to delete some colonies!";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public CheckForAutoDeletesCommand(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @Override
+    public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final EntityPlayer player)
+    {
+        return colony.getPermissions().getRank(player).equals(Rank.OWNER);
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        final List<Colony> colonies = ColonyManager.getColonies();
+
+        boolean colonyWasDeleted = false;
+
+        for (int index = 0; colonies.size() - 1 >= index; index++)
```

Comment:
I think we should handle that more seriously.
There should be a "this will delete x colonies are you really sure you want to do that" ?